historical/common/dynamodb.py

```diff
@@ -199,7 +199,7 @@ def deserialize_current_record_to_current_model(record, current_model):
     :return:
     """
     # Was the item in question too big for SNS? If so, then we need to fetch the item from the current Dynamo table:
-    if record.get("sns_too_big"):
+    if record.get("event_too_big"):
```

Comment:
Could this pulled out into a `CONSTANT` variable? In case it needs to be change in the future or multiple values could be accepted? i.e. migrating from one to another?

Reply:
Will do.